io_scene_godot/converters/mesh.py

```diff
@@ -89,6 +93,20 @@ def get_modifier_armature_data(mesh_object):
     return None
 
 
+def export_mesh_link_material(escn_file, export_settings, mesh_object,
+                              gd_node):
+    """Export the active material link to object"""
+    if mesh_object.material_slots:
+        active_idx = mesh_object.active_material_index
+        material = mesh_object.material_slots[active_idx].material
+        if material is not None:
+            gd_node['material/0'] = export_material(
```

Comment:
Ah, this is what is causing it. It may be worth checking how it used to be handled, because this doesn't support multiple-material objects.